CMakeLists.txt

```diff
@@ -1,30 +1,33 @@
 cmake_minimum_required(VERSION 3.15)
 
-project(boost-asio-windows-sspi CXX)
+project(boost-windows-sspi CXX)
 
 set(CMAKE_CXX_STANDARD 11)
 
+enable_testing()
```

Comment:
`enable_testing` enables tests unconditionally, however, you might want to use the standard CMake approach:
```
include(CTest)
if(BUILD_TESTING)
    enable_testing()
    add_subdirectory(tests)
endif()
```

Which allows users to disable building tests of your library. This is useful for users consuming the library e.g. via `vcpkg`, so that they don't need to distribute your test binaries in their `vcpkg export`.

Reply:
Good point. I have considered making the build more customizable in different ways, but I might as well look into that right now. Thanks.